app/scripts/modules/core/src/pipeline/config/stages/wercker/werckerStage.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
```

Comment:
I suggest renaming this file from `.js` to `.ts` and then using the lint output generated by `yarn build`, `yarn tslint` or `yarn start` to guide you through linting your code.

Reply:
It may not be just simply renaming all the js to ts as we are using jeknins trigger and stage as examples to create wercker's, and those module.js files
are using angular.module('spinnaker.core.pipeline.config.trigger.jenkins'...) to config the module. It results in tsModuleNotFoundError if renaming the file from js to ts without changing the codes to, for example, travis ways of module.config.

I did run yarn run build without error in the new codes.

Reply:
Yup that makes sense - using the travis trigger as an example would be a good way to go!